### DISCUSSION TITLE
server port?

### DISCUSSION THREAD

**Discussion by andysay** [nuxt/nuxt#18204] [Questions] [answered]
how set custom server port on build mode ? config not support this

  server: {
    port: 8000 // default: 3000
  }

**Comment by HealerNguyen** [comment, marked as answer]
u can using cross-env and simply add your port 
![image](https://user-images.githubusercontent.com/29653304/140887630-3eabca12-9407-4cea-a99c-036c5034bd62.png)

**Reply by andysay** [reply]
Build , start will work on 3000 I want change

**Reply by HealerNguyen** [reply]
> Build , start will work on 3000 I want change

![image](https://user-images.githubusercontent.com/29653304/140893626-4f452bb4-6697-4f53-98ac-e30bf440a709.png)
that is my setting and my port is change
